addons/mail/static/src/discuss/channel_invitation.js

```diff
@@ -93,9 +93,11 @@ export class ChannelInvitation extends Component {
         if (this.props.thread.type === "chat") {
             const partners_to = [
                 this.store.self.id,
-                this.props.thread.chatPartnerId,
                 ...this.state.selectedPartners.map((partner) => partner.id),
             ];
+            if (this.props.thread.chatPartnerId != this.store.self.id) {
```

Comment:
Be careful, you forgot an `=` 👀
```suggestion
            if (this.props.thread.chatPartnerId !== this.store.self.id) {
```